tests/pr_test.go

```diff
@@ -8,17 +8,23 @@ import (
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
+// Restricting due to limited availability of BYOK in certain regions
+const regionSelectionPath = "../common-dev-assets/common-go-assets/icd-region-prefs.yaml"
+
 // Use existing resource group
-const resourceGroup = "geretain-test-resources"
+const resourceGroup = "geretain-test-mongo"
+
 const defaultExampleTerraformDir = "examples/default"
+const autoscalingExampleTerraformDir = "examples/autoscale"
+const completeExampleTerraformDir = "examples/complete"
 
 func TestRunDefaultExample(t *testing.T) {
 	t.Parallel()
 
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
 		Testing:       t,
 		TerraformDir:  defaultExampleTerraformDir,
-		Prefix:        "mod-template",
+		Prefix:        "mongodb",
```

Comment:
Probably add a suffix to this so that we know which unit test the resource came from, see how the upgrade adds a "-upg" to the end, something like that.